go-controller/pkg/ovn/kubevirt_test.go

```diff
@@ -825,10 +825,20 @@ var _ = Describe("OVN Kubevirt Operations", func() {
 				}
 
 				if t.podName != "" {
+					pod, err := fakeOvn.fakeClient.KubeClient.CoreV1().Pods(t.namespace).Get(context.TODO(), t.podName, metav1.GetOptions{})
+					Expect(err).NotTo(HaveOccurred())
+					pod.Status.Phase = corev1.PodSucceeded
+					_, err = fakeOvn.fakeClient.KubeClient.CoreV1().Pods(t.namespace).UpdateStatus(context.TODO(), pod, metav1.UpdateOptions{})
```

Comment:
nit. does `UpdateStatus` expect to pass at first attempt itself ? or do we need multiple attempts with `eventually` here ?

Reply:
This is unit testing, fakeClient has no conflicts mechanism, this is good enough.